site/en/docs/webstore/i18n/index.md

```diff
@@ -1,132 +1,124 @@
+| fi          | Finnish                               |
+| fil         | Filipino                              |
+| fr          | French                                |
+| gu          | Gujarati                              |
+| he          | Hebrew                                |
+| hi          | Hindi                                 |
+| hr          | Croatian                              |
+| hu          | Hungarian                             |
+| id          | Indonesian                            |
+| it          | Italian                               |
+| ja          | Japanese                              |
+| kn          | Kannada                               |
+| ko          | Korean                                |
+| lt          | Lithuanian                            |
+| lv          | Latvian                               |
+| ml          | Malayalam                             |
+| mr          | Marathi                               |
+| ms          | Malay                                 |
+| nl          | Dutch                                 |
+| no          | Norwegian                             |
+| pl          | Polish                                |
+| pt_BR       | Portuguese (Brazil)                   |
+| pt_PT       | Portuguese (Portugal)                 |
+| ro          | Romanian                              |
+| ru          | Russian                               |
+| sk          | Slovak                                |
+| sl          | Slovenian                             |
+| sr          | Serbian                               |
+| sv          | Swedish                               |
+| sw          | Swahili                               |
+| ta          | Tamil                                 |
+| te          | Telugu                                |
+| th          | Thai                                  |
+| tr          | Turkish                               |
+| uk          | Ukrainian                             |
+| vi          | Vietnamese                            |
+| zh_CN       | Chinese (China)                       |
+| zh_TW       | Chinese (Taiwan)                      |
+
+## Localize your extension listing
+
+To localize your extension's listing, you first localize the name and description. Then, after uploading
+your localized extension, you can edit your extension's listing to specify localized versions of your extension's
 detailed description and screenshots.
 
-### Name and description
+### Name and description {: #localize-manifest }
+
+You localize the [name][name] and [description][description] by modifying `manifest.json` and providing a `_locales`
+directory in your extension's ZIP file. Specifically, you need to:
 
-You localize the name and description by modifying `manifest.json` and providing a `_locales`
-directory in your app's ZIP file. Specifically, you need to:
+- Modify the `"name"` and `"description"` manifest fields.
+- Add a `"default_locale"` field.
```

Comment:
I'm thinking it may be a little better to link `name` and `description` in the bulleted list, add a new link on `default_locale` to /docs/extensions/mv3/manifest/default_locale/, and to link `manifest.json` in the intro to /docs/extensions/mv3/manifest/. WDYT?